.changes/unreleased/tinymce-TINY-10306-2025-09-18.yaml

```diff
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Fixed
+body: The cursor could get stuck around an absolute CEF element when navigating using arrow keys.
```

Comment:
I think we just need one changelog item then you can just link them in jira if they are related.